src/lib/eina/eina_thread.c

```diff
@@ -228,30 +130,46 @@ EAPI Eina_Bool
 eina_thread_cancel(Eina_Thread t)
 {
    if (!t) return EINA_FALSE;
+   #ifndef _WIN32
    return pthread_cancel((pthread_t)t) == 0;
+   #else
+   return _eina_thread_cancel(t);
+   #endif
```

Comment:
Why not use `_eina_thread_cancel` in both cases?

Reply:
Ok, i'll change it. In eina_thread.c I changed as little as possible.